src/components/HomePage/Event.js

```diff
@@ -20,32 +19,51 @@ const useStyles = makeStyles(theme => ({
 		alignItems: 'center',
 		marginTop: '1ex',
 		'& > :nth-child(1)': {
-			marginRight: '1.5em'
+			marginRight: '1em'
+		},
+		[theme.breakpoints.up('sm')]: {
+			'& > :nth-child(1)': {
+				marginRight: '1.5em'
+			}
 		}
```

Comment:
Consider

```suggestion
		'& > :nth-child(1)': {
			marginRight: '1em'
			[theme.breakpoints.up('sm')]: {
				marginRight: '1.5em'
			}
		}
```

also consider using `:nth-child(odd)`, considering we want margin-right on subtitles 1, 3, 5, … (though it doesn't currently matter since we never have 4 subtitles)